test/Directory.Build.targets

```diff
@@ -1,12 +1,13 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project>
-  <Import Project="..\Directory.Build.targets" />
 
-  <!-- forces the version in sub dependencies -->
-  <!-- xunit.assert has this dependency, so applying to all test projects-->
-  <ItemGroup>
-    <PackageReference Condition="'$(IsTestProject)' == 'true'" Include="System.Net.Http" Version="4.3.4" />
-    <PackageReference Condition="'$(IsTestProject)' == 'true'" Include="System.Security.Cryptography.X509Certificates" Version="4.3.0" /> 
-    <PackageReference Condition="'$(IsTestProject)' == 'true'" Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
We had to add other package references to ensure no issues in CG. Has something changed since then?